src/components/DateTimePicker/Internal/OcPicker.types.ts

```diff
@@ -765,6 +821,15 @@ export type OcPickerSharedProps<DateType> = {
    * The picker tab index.
    */
   tabIndex?: number;
+  /**
+   * The 'Today' button props.
+   */
+  todayButtonProps?: ButtonProps;
+  /**
+   * The current day appears to be active.
```

Comment:
appears to be = "is" hopefully

Reply:
Wanted to disambiguate selected from active, active is a passive visual state. I'll need to revisit this naming.